src/css/components/loading.scss

```diff
@@ -2,25 +2,66 @@
 $z-foreground: 200;
 
 .loading {
```

Comment:
Absolutely positioned the block loader to get it to properly display full height of parent